test/examples/uct_hello_world.c

```diff
@@ -1,17 +1,611 @@
 /**
-* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
-*
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
 * See file LICENSE for terms.
 */
 
-#define HAVE_CONFIG_H /* Force using config.h, so test would fail if header
-                         actually tries to use it */
+#include "ucx_hello_world.h"
+
 #include <uct/api/uct.h>
-#include <stdio.h>
+#include <ucs/type/status.h>
+#include <ucs/debug/log.h>
```

Comment:
Please, use system `assert` because this is user application. `ucs_assert` is internal UCX feature and we don't require the user to use it.